README.md

```diff
@@ -16,35 +16,54 @@ The structure of this file is defined in [assets/schema_input.json](assets/schem
 
 # Parameters
 
-The main parameters are `--input` as defined above and `--output` for specifying the output results directory.
+## Mandatory
+
+The mandatory parameters are as follows:
+
+- `--input` : a URI to the samplesheet as specified in the [Input](#input) section.
+- `--output` : to specify the output results directory.
+- `--kraken2_db /path/to/kraken2database` : to specify the directory to the Kraken2 database
+- `--bracken_db /path/to/brackendatabase` : to specify the directory to the Bracken database
+
+## Optional
 
 Additionally, you may wish to provide:
 
-`-profile singularity` to specify the use of singularity containers
+### SpeciesAbundance Parameters
 
-`-r [branch]` to specify which GitHub branch you would like to run
+- `--taxonomic_level` : to specify the taxonomic level of the bracken abundance estimation.
+  - Must be one of 'S'(species)(default), 'G'(genus), 'O'(order), 'F'(family), 'P'(phylum), or 'K'(kingdom)
 
-`--kraken2_db /path/to/kraken2database`
+### Other Parameters
 
-`--bracken_db /path/to/brackendatabase`
+- `-profile` : to specify which profile to use (ex: `-profile singularity`)
+- `-r [branch]` : to specify which GitHub branch you would like to use
 
 Other parameters (defaults from nf-core) are defined in [nextflow_schema.json](nextflow_schmea.json).
 
 # Running
 
-To run the pipeline, please do:
+## Test Data
+
+To run the pipeline, please run:
 
 ```bash
-nextflow run phac-nml/speciesabundance -profile singularity -r dev -latest --input /path/to/samplesheet.csv --outdir results
+nextflow run phac-nml/speciesabundance -profile singularity -r dev -latest --input /path/to/samplesheet.csv --kraken2_db /path/to/kraken2_db -- bracken_db /path/to/bracken_db --outdir results
```

Comment:
Could this example line have the paths to the test data in `tests/data`? Or perhaps have a second line showing how to run it with the provided test data. Especially considering it's under a header called `Test Data`.

Reply:
I thought about this for a while due to the issue of not being able to add an http:// URL to a directory of files stored in the GitHub repo. Instead I settled for removing the code to run with provided data and re-organized the test profile option. [dd6bd99](https://github.com/phac-nml/speciesabundance/pull/11/commits/dd6bd99923b39487ec5bca0d8d976dfea9b2e88c)